index.html

```diff
@@ -0,0 +1,230 @@
+
+                <h2>Multi- Post Stories</h2>
+
+                <div class="details">
+                    <span id="canopy">CANOPY</span>
+                    <img src="images/Count.png" alt="greydot">
+                    <span class="pair" id="wider">Back End Dev</span>
+                    <img src="images/Count.png" alt="greydot">
+                    <span class="pair">2015</span>
+                </div>
+
+                <p>A daily selection of privately personalized reads;
+                    no accounts or sign-ups required.</p>
+
+                <ul>
+                    <li class="one">HTML</li>
+                    <li class="two">CSS</li>
+                    <li class="three">Javascript</li>
+                </ul>
+
+                <button>See project</button>
+
+            </div>
+        </div>
+
+        <!-- CARD 3 -->
+        <div class="card">
+
+            <div class="cardimage">
+                <img src="images/Snapshoot(3).png" alt="">
+            </div>
+
+            <div class="workdescription">
+
+                <h2>Tonic</h2>
+
+                <div class="details">
+                    <span id="canopy">CANOPY</span>
+                    <img src="images/Count.png" alt="greydot">
+                    <span class="pair" id="wider">Back End Dev</span>
+                    <img src="images/Count.png" alt="greydot">
+                    <span class="pair">2015</span>
+                </div>
+
+                <p>A daily selection of privately personalized reads;
+                    no accounts or sign-ups required.</p>
+
+                <ul>
+                    <li class="one">HTML</li>
+                    <li class="two">CSS</li>
+                    <li class="three">Javascript</li>
+                </ul>
+
+                <button>See project</button>
+
+            </div>
+        </div>
+
+        <!-- CARD 4 -->
+        <div class="card">
+
+            <div class="cardimage">
+                <img src="images/Snapshoot(4).png" alt="">
+            </div>
+
+            <div class="workdescription">
+
+                <h2>Multi- Post Stories</h2>
+
+                <div class="details">
+                    <span id="canopy">CANOPY</span>
+                    <img src="images/Count.png" alt="greydot">
+                    <span class="pair" id="wider">Back End Dev</span>
+                    <img src="images/Count.png" alt="greydot">
+                    <span class="pair">2015</span>
+                </div>
+
+                <p>A daily selection of privately personalized reads;
+                    no accounts or sign-ups required.</p>
+
+                <ul>
+                    <li class="one">HTML</li>
+                    <li class="two">CSS</li>
+                    <li class="three">Javascript</li>
+                </ul>
+
+                <button class="seeproject">See project</button>
+
+            </div>
+        </div>
+
+
+    </section>
+
+    <article class="about-section">
```

Comment:
- [ ] Kindly use a `section` element to wrap the About section. This is best for HTML semantics.

Reply:
@Kamba56  Noted with thanks.